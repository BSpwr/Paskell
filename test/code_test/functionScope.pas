program functionScope;

// global vars
var g : string = 'Im a global!';

procedure hi;
begin
    writeln(g);
    g := 'Global set from function';
end;

begin
hi;
writeln(g);
end.