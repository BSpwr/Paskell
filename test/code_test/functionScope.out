Im a global!
Global set from function